src/saltedhash.cpp

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) 2019 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <random.h>
+#include <saltedhash.h>
```

Comment:
2ba96b0438fe734b788e56ba881303c84a3f249d
nit:
```suggestion
#include <saltedhash.h>

#include <random.h>

#include <limits>
```

Reply:
Added `#include <limits>`. `random.h` is one of our headers, so it should be grouped with `saltedhash.h`.

Reply:
I mean the common pattern in the repo for `somefeature.cpp`:
```
// Copyright...

#include <somefeature.h>

#include <anotherheader.h>
#include <otherheader.h>
//...
```

nm, this is not documented :)